dashboard/tabs/trends.py

```
@@ -38,16 +42,16 @@ def _render_timeseries(
     tick_labels  = [pd.Timestamp(d).strftime("%Y-%m-%d") for d in unique_dates]
     n = len(present_metrics)
 
-    # Lay out metrics in a 2-column grid (ceil(n/2) rows × 2 cols)
-    n_cols = min(n, 2)
+    # Lay out metrics in a 3-column grid (ceil(n/3) rows × 3 cols)
```

Ruff (0.15.14)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)